src/main/kotlin/racingcar/domain/move/RandomMovingStrategy.kt

```diff
@@ -0,0 +1,12 @@
+package racingcar.domain.move
+
+import kotlin.random.Random
+
+private const val RANDOM_NUMBER_BOUND = 10
+private const val MOVING_THRESHOLD = 4
+
+class RandomMovingStrategy : MovingStrategy {
+    override fun canMove(): Boolean {
+        return Random.nextInt(RANDOM_NUMBER_BOUND) >= MOVING_THRESHOLD
+    }
+}
```

Comment:
```suggestion
private val RANDOM_NUMBER_RANGE = 0..9
private const val MOVING_THRESHOLD = 4

class RandomMovingStrategy : MovingStrategy {
    override fun canMove(): Boolean = RANDOM_NUMBER_RANGE.random() >= MOVING_THRESHOLD
}
```
0..9 사이의 값을 10으로 표현하는 것보다, intRange를 통하여 가독성 있게 표현이 가능합니다!